content/docs/getting-started/accounts.md

```diff
@@ -35,14 +35,18 @@ Your cloud.gov account requires setting up both a password and an authentication
 
 #### If you can't access your token codes
 
-If you lose access to your authentication application (such as if you lose your phone), email [cloud.gov support](/help/) so that we can allow you to set up a new one. We'll follow this process to mitigate the risk of requests from compromised email addresses:
+If you need to set up a new authentication application, such as if you lose your phone that had your authentication application, [email cloud.gov support](mailto:cloud-gov-support@gsa.gov?subject=MFA%20reset&body=I%20need%20to%20set%20up%20a%20new%20authentication%20application.%20I%20understand%20this%20means%20the%20contents%20of%20my%20sandbox%20space%20will%20be%20deleted%20if%20I%20have%20one,%20and%20that%20you%20will%20remove%20my%20permissions%20to%20other%20spaces%20and%20orgs.) so that we can allow you to set up a new one. We'll follow this process to mitigate the risk of requests from compromised email addresses:
```

Comment:
It reads a bit awkwardly with `that had your authentication application`... Can we remove that? Otherwise I'm good to go with this.

Reply:
Done, thanks!